contracts/CreateAccount.sol

```diff
@@ -160,22 +160,38 @@ contract CreateAccount is FraudProofHelpers {
         // Assuming Reddit have run rollup.sol::createPublickeys
         ValidatePubkeyAvailability(_accountsRoot, _to_pda_proof, _tx.toIndex);
 
-        // accountProofs.to.accountIP.account should to be a zero account
-        bool result = ValidateZeroAccount(accountProofs.to.accountIP.account);
-        if (result == false) {
-            return ("", "", "", 0, false);
+        // Validate we are creating on a zero account
+        if (
+            !merkleUtils.verifyLeaf(
+                _balanceRoot,
+                keccak256(abi.encode(0)), // default hash 0
+                to_account_proof.accountIP.pathToAccount,
+                to_account_proof.siblings
+            )
```

Comment:
There's no actual zero account that looks like`{fromIndex: 0, toIndex:0 ...}`, we only have zero account leaf. So here's how we validate the proof.

Reply:
Sounds good, just for the sake of saving some gas and not needing to hardcode things, can you use [this](https://github.com/thehubbleproject/contracts/blob/master/contracts/MerkleTreeUtils.sol#L9) instead of `keccak256(abi.encode(0))`?

Reply:
So just merkleUtils.defaulutHashes(0) should work